site/app/templates/grading/electronic/ta_status/StatusBase.twig

```diff
@@ -208,7 +208,7 @@
                     <li>Grade Inquiries</li>
                 {% endif %}
             </ul>
-            {% if core.getUser().getGradingRegistrationSections()|length != 0 %}
+            {# {% if core.getUser().getGradingRegistrationSections()|length != 0 %}
```

Comment:
why is this commented out?

Reply:
When the button is clicked, it goes to a frog robot page - the button only appears when there are no students that have been graded, and I think it was designed for the old TA grading page (where you could call it with no parameters and it would give a ungraded student, although even without the frog robot the appear logic looks wrong too)
We talked about it a bit during the meeting and for now decided that we don't want the button (so maybe it should be deleted instead of commented out)

Reply:
Yeah, the main thing here for commenting out code is to give context for why it's commented out, when it might be uncommented out, link to a GH issue maybe, etc. Assume the student next semester that looks at this is not going to have the benefit of having been at the meeting that this was discussed.

However, if it's broken / dead code we no longer want / need, better to remove it altogether! 😄